crates/algod_client/src/models/box_model.rs

```diff
@@ -19,11 +19,11 @@ pub struct Box {
     /// The round for which this information is relevant
     #[serde(rename = "round")]
     pub round: u64,
-    /// \[name\] box name, base64 encoded
+    /// The box name, base64 encoded
     #[serde_as(as = "serde_with::base64::Base64")]
     #[serde(rename = "name")]
     pub name: Vec<u8>,
```

Comment:
FYI I have a generator change to properly handle Base64 values in the OAS for endpoints that support msgpack. They are all bytes values and only base64 when using JSON.

It's unrelated to this PR, however figured it was worth a heads up in case you notice issues.

 however a heads up in case you hit issues.